prow/plugins/label/label.go

```diff
@@ -213,6 +223,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	mID, err := strconv.Atoi(maintainersID)
+	if err != nil {
+		log.WithError(err).Errorf("Couldn't convert the maintainers team ID parameter to int %v", maintainersID)
+	}
+	milestoneMaintainers, err := gc.ListTeamMembers(mID)
+	maintainersMap := map[string]bool{}
+	if err == nil {
+		for _, person := range milestoneMaintainers {
+			maintainersMap[person.Login] = true
+		}
+	} else {
+		log.WithError(err).Errorf("Failed to list the teammembers for the milestone maintainers team")
+	}
+
+	for _, statusMatch := range statusMatches {
+		status := strings.TrimSpace(statusMatch[1])
+		sLabel, validStatus := statusMap[status]
+		if validStatus {
+			_, ok := maintainersMap[ae.login]
+			if ok {
+				if err := gc.AddLabel(ae.org, ae.repo, ae.number, sLabel); err != nil {
+					log.WithError(err).Errorf("Github failed to add the following label: %s", sLabel)
```

Comment:
Propagate up these errors rather than logging them.

Reply:
Same response as above.  I can't return early because someone could in theory write `/status approved-for-milestone\n/sig node`, and we'd still want the node part to complete.

Reply:
Ok